### ISSUE TITLE
domain request: danbooru and rule34 and adblocker  fix for toonily

### ISSUE THREAD

**Issue by givena** [avluis/Hentoid#804]
hello, can you please add those domains and If you can block the domain of the new popup ads on toonily

**Comment by RobbWatershed** [comment]
> hello, can you please add those domains 

TL;DR We can't with the current state of the app - we suggest using a booru reader app such as flexbooru, mbooru or booru nav.

The entire app revolves around the idea that content are books with a set number of pages that are read from page 1 to page X. Now boorus are breaking all of that and feature a ton of pictures, every single one of them tagged with specific metadata. That's quite a switch to make !

Supporting them on Hentoid would require search and reading features to be able to navigate through pictures grouped and filtered by metadata, which is totally feasible, but heavy in terms of development.

> and If you can block the domain of the new popup ads on toonily

Today's release (v1.14.5) should remove all ad hosts on Toonily, Manwha and Luscious.
